scripts/integration/prometheus/compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  influxdb:
+    image: docker.io/influxdb:${PROMETHEUS_VERSION}
+    environment:
+    - INFLUXDB_REPORTING_DISABLED=true
+  influxdb-tls:
+    image: docker.io/influxdb:${PROMETHEUS_VERSION}
+    environment:
+    - INFLUXDB_REPORTING_DISABLED=true
+    - INFLUXDB_HTTP_HTTPS_ENABLED=true
+    - INFLUXDB_HTTP_BIND_ADDRESS=:8087
+    - INFLUXDB_BIND_ADDRESS=:8089
+    - INFLUXDB_HTTP_HTTPS_CERTIFICATE=/etc/ssl/intermediate_server/certs/localhost-chain.cert.pem
+    - INFLUXDB_HTTP_HTTPS_PRIVATE_KEY=/etc/ssl/intermediate_server/private/localhost.key.pem
+    volumes:
+    - ../../../tests/data/ca:/etc/ssl:ro
```

Comment:
Noting that I'm real annoyed that we involve influx in our prometheus tests...

Reply:
Ah, if it's just for remote write - it is supported, I think it's just behind a config flag or something similar 👍

Reply:
Regardless, unrelated to the changes here